### PR TITLE
docs: Update team working flow (#206)

### PR DESCRIPTION
**Pull request by Oleksandr123234**
## Description
This change contains updates to the KubeRocketCI working flow documentation.
Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [*] Documentation update

## Checklist:
- [*] I have performed a self-review of my own code
- [*] I have commented my code, particularly in hard-to-understand areas
- [*] I have made corresponding changes to the documentation
- [*] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [*] Pull Request contain one commit. I squash my commits

### CODE REVIEW

File: docs/developer-guide/edp-workflow.md [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
 
-  a. Voting in Gerrit.
+  a. Creating pull requests in GitHub.
 
   b. Resolution of Merge Conflict.
 
   c. Comments resolution.
 
-  d. One Jira task should have one Merge Request (MR). If there are many changes within one MR, add the next patch set to the open MR by selecting the _Amend commit_ check box.
+  d. One Jira task should have one Merge Request (MR) if it doesn't change multiple operators. If there are many changes within one MR, amend the commit.
```
Comment (NikolayMarusenko) [inline docs/developer-guide/edp-workflow.md:17]: are we using Jira in this document?
Maybe we're ready to move to GitHub release?

File: docs/developer-guide/edp-workflow.md [hunk 2]
```diff
@@ -1,30 +1,30 @@
...
+1. It is highly recommended to become familiar with the GitHub flow. For details, please refer to the [GitHub official documentation](https://docs.github.com/en/get-started/using-github/github-flow) and pay attention to the main points:
 
-  a. Voting in Gerrit.
+  a. Creating pull requests in GitHub.
 
   b. Resolution of Merge Conflict.
 
   c. Comments resolution.
 
-  d. One Jira task should have one Merge Request (MR). If there are many changes within one MR, add the next patch set to the open MR by selecting the _Amend commit_ check box.
```
Comment (NikolayMarusenko) [inline docs/developer-guide/edp-workflow.md:17]: Pull reqyests (PR)?

File: docs/developer-guide/edp-workflow.md (hunk 1 above)
Comment (Oleksandr123234) [reply docs/developer-guide/edp-workflow.md]: fixed

File: docs/developer-guide/edp-workflow.md (hunk 2 above)
Comment (Oleksandr123234) [reply docs/developer-guide/edp-workflow.md:17]: fixed

**Review by SergK — APPROVED** [review]

**Pull request merged**